src/utils/analytics/events.ts

```diff
@@ -0,0 +1,18 @@
+export enum HotspotEvents {
+  ADD_GATEWAY_INITIATED = 'AddGateway Initiated',
+  ADD_GATEWAY_SUBMITTED = 'AddGateway Submitted',
+  ASSERT_LOCATION_INITIATED = 'AssertLocation Initiated',
+  ASSERT_LOCATION_SUBMITTED = 'AssertLocation Submitted',
+  DEVICE_TRANSFER_INITIATED = 'DeviceTransfer Initiated',
+  DEVICE_TRANSFER_SUBMITTED = 'DeviceTransfer Submitted',
+  BLUETOOTH_SCAN_STARTED = 'Bluetooth Scan Started',
+  BLUETOOTH_SCAN_FINISHED = 'Bluetooth Scan Finished',
```

Comment:
What's the difference between finished and succeeded?

Reply:
Essentially they are same. Let me standardize them.